docs/resources/ngwaf_datadog_alert.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: "fastly"
+page_title: "Fastly: ngwaf_datadog_alert"
+sidebar_current: "docs-fastly-resource-ngwaf-datadog-alert"
+description: |-
+  Provides a Fastly Next-Gen WAF Redaction for a Workspace
+---
+
+# ngwaf_datadog_alert
+
+Provides a Fastly Next-Gen WAF Datadog Alert, which provides a connection to Datadog.
+
+## Example Usage
+
+Basic usage:
+
+```terraform
+resource "fastly_ngwaf_datadog_alert" "demo_datadog_alert" {
+  description                  = "Some Description"
+  integration_key              = "123456789"
+  integration_site             = "us1"
+  workspace_id                 = fastly_ngwaf_workspace.demo.id
+}
+```
+
+## Import
+
+Fastly Next-Gen WAF Datadog Alerts can be imported using their ID and the ID of the workspace they belong to.
+
+```sh
+$ terraform import fastly_ngwaf_datadog_alert.example <workspace_id>/<alert_id>
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `integration_key` (String) The Datadog integration key.
+- `integration_site` (String) The Datadog site.
+- `workspace_id` (String) The id of the workspace this alert belongs to.
+
+### Optional
+
+- `description` (String) User-submitted description of the integration
```

Comment:
Please replace attribute descriptions with the ones from the API docs that will be published next week (https://branch-apidocs-console-1533-ngwaf-functional-parity.developer.fastly.com/documentation/reference/api/ngwaf/workspace-alerts/) - for example this should be "An optional description for the alert.